hotels.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Гостиницы. HTML Academy: Седона. </title>
+  </head>
+  <body>
+
+    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="" width="" height="" alt="Логотип SITY OF SEDONA">
+        </a>
+        <ul>
+          <li><a href="blank.html">Информация</a></li>
+          <li><a href="blank.html">Фото и видео</a></li>
+          <li><a href="blank.html">Карта штата</a></li>
+          <li><a href="hotels.html">Гостиницы</a></li>
```

Comment:
```suggestion
          <li><a>Гостиницы</a></li>
```

Страница не должна на себя ссылаться.